stripper/ze_paranoid_rezurrection_csgo1.cfg

```diff
@@ -1,50 +1,310 @@
-; "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 1.
+// "ze_paranoid_rezurrection_csgo1" config, made by "Berke" "STEAM_1:0:95142811", version 2.
 
-; Changes from "GFL".
-; Fix "game_player_equip" dropping weapons and fix kevlar buying.
+// Delete low buy time, extend height of spawn trigger and fix zombies not getting teleported.
 modify:
 {
 	match:
 	{
-		"classname" "game_player_equip"
-		"targetname" "darP90"
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "protect_player,AddOutput,solid 2,,1"
+		"OnMapSpawn" "protect_playerRunScriptCodeself.SetSize(Vector(-1600, -1178.5, -184.017), Vector(1600, 1178.5, 616));1"
+		"OnMapSpawn" "playerRunScriptCodeif (self.GetTeam() == 2 && self.GetHealth()) { self.SetOrigin(Vector(-12408, -8140, 318)); self.SetAngles(0, -89, 0); }301"
+	}
+	delete:
+	{
+		"OnMapSpawn" "consolaCommandmp_buytime 22-1"
+		"OnMapSpawn" "protect_playerEnable5-1"
+	}
+}
+
+// Is this needed? Not referenced in VMF or map vscript, only seems to be for spawning players but it isn't really needed.
+filter:
+{
+	"classname" "game_player_equip"
+	"targetname" "darP90"
+}
+
+// Fix spawn bugs.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "protect_player"
 	}
 	replace:
 	{
-		"spawnflags" "2"
+		"StartDisabled" "0"
+	}
+	delete:
+	{
+		"model" "*386"
+		"OnEndTouchAll" "!activatorSetDamageFilter0-1"
 	}
 	insert:
 	{
-		"item_assaultsuit" "1"
+		"OnStartTouch" "!activator,Color,255 255 255,,-1"
+		"OnEndTouch" "!activator,SetDamageFilter,,,-1"
 	}
 }
 
-; Delete low buy time.
+// Fix "Jukebox".
 modify:
 {
 	match:
 	{
-		"classname" "logic_auto"
+		"classname" "trigger_multiple"
+		"targetname" "spawn_pushHumanos2x"
+	}
+	insert:
+	{
+		"OnStartTouch" "w_djRunScriptCodeself.SetOrigin(Vector(-12436, -9413.8, 740.318));1"
+	}
+}
+
+// Fix winner status not resetting.
+add:
+{
+	"classname" "game_round_end"
+	"targetname" "map1winnerstatusreset1"
+	"OnRoundEnded" "player,RunScriptCode,WINNER <- 0;,,1"
+	"OnRoundEnded" "!self,Kill,,,1"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "har2"
+	}
+	insert:
+	{
+		"OnStartTouch" "map1winnerstatusreset1,Kill,,,1"
+		"OnStartTouch" "player,RunScriptCode,WINNER <- 0;,,1"
```

Comment:
Make sure to use escape characters rather than commas when adding script outputs. This is actually the reason vscript-supporting games switched to the escape character, otherwise there would be ambiguity between a script comma and an output separating comma.

Reply:
Shouldn't be needed in this case since parameter doesn't use any commas, I tested it, works fine, more readable in my opinion. I prefer only using it when parameter uses it

Reply:
Whether or not you're using commas in the script parameter, there is still ambiguity as to what the separator commas should be a part of. Parsing screwing up at the end won't inherently guarantee visible breakage, and I'm not really keen on digging through all the output parsing code to guarantee it's sane in this case.